admin-dev/themes/default/template/helpers/list/list_content.tpl

```diff
@@ -26,7 +26,7 @@
 <tbody>
 {if count($list)}
 {foreach $list AS $index => $tr}
-	<tr{if $position_identifier} id="tr_{$position_group_identifier}_{$tr.$identifier}_{if isset($tr.position['position'])}{$tr.position['position']}{else}0{/if}"{/if} class="{if isset($tr.class)}{$tr.class}{/if} {if $tr@iteration is odd by 1}odd{/if}"{if isset($tr.color) && $color_on_bg} style="background-color: {$tr.color}"{/if} >
```

Comment:
I guess the odd class is removed here because now all the ros look the same right? I don't know if it's relevant to keep it, besides with modern CSS you can use selector like `tr:nth-child(even)` `tr:nth-child(odd)` so even a design that wants to customize this could rely on pure CSS selectors instead of this class

Reply:
I agree!

Reply:
Yes, I removed this because of the new look of the BO. Indeed, it can now be easily done with CSS only.